src/content/self-hosted/manage/troubleshooting.mdx

```diff
@@ -5,6 +5,45 @@ sidebar_label: Troubleshooting
 id: troubleshooting
 ---
 
+## How to extract logs from Okteto when asking for help
+
+When asking for assistance from Okteto Support, please use the [`okteto logs` command](reference/okteto-cli.mdx#logs) to fetch the logs from your Development Environment.
+
+## Understanding the state of your app: why endpoints aren't present in the CLI or UI
+
+Endpoint links are not present within Okteto if there are no services actively running. If your endpoints are missing, consider the following states and their implications:
+
+#### Progressing: 
+* Description: Your deployment is in the process of being rolled out
+* Possible Causes:
+  * Your service is still being started
+  * There are pending updates or new deployments
```

Comment:
Related to previous comment, I think we also check for healthchecks, but not sure